plugins/node/opentelemetry-instrumentation-connect/src/instrumentation.ts

```diff
@@ -180,4 +212,33 @@ export class ConnectInstrumentation extends InstrumentationBase<Server> {
       return original.apply(this, args as UseArgs2);
     };
   }
+
+  public _patchHandle(original: Server['handle']): Server['handle'] {
+    const instrumentation = this;
+    return function (
+      this: Server,
+      req: PatchedRequest,
+      res: ServerResponse,
+      out?: Function
+    ): ReturnType<Server['handle']> {
+      addNewStackLayer(req);
+
+      function completeStack() {
+        req[_LAYERS_STORE_PROPERTY].pop();
+      }
+      const done = out
+        ? instrumentation._patchOut(out as NextFunction, completeStack)
+        : undefined;
+
+      return original.apply(this, [req, res, done as Function]);
```

Comment:
not sure if it is relevant but this may result in an visible side effect.
inside the original function `arguments.length` is now always 3, it was potentially 2 before.
Also any extra argument after `out` is lost.